### ISSUE TITLE
why don't spotify player status in MacOS be prefered to use osascript?

### ISSUE THREAD

**Issue by arslan2012** [powerline/powerline#1815]
in /powerline/segments/common/players.py line 374
```
if 'dbus' in globals() or not sys.platform.startswith('darwin'):
```
this means that even in macOS dbus is preffered way.
but in macOS dbus-python is very buggy, dbus module is buggy no matter installed by brew or ports.
on the contrary, osascript is preinstalled on OS X systems and rarely have issue.

I think that line should be something like
```
if 'dbus' in globals() and not sys.platform.startswith('darwin'):
```

or even just
```
if not sys.platform.startswith('darwin'):
```
because linux system has to use dbus anyways.

**Comment by ZyX-I** [comment]
What do you mean by “buggy”? Dbus is supposed to be faster because it does not need to spawn processes.

**Comment by arslan2012** [comment]
![example](https://user-images.githubusercontent.com/8698442/29027219-4fa5f56c-7bb2-11e7-84df-d0fd0f6f425a.png)

**Comment by arslan2012** [comment]
I have macOS 10.12.6

I installed dbus with brew install dbus. dbus-python by pip install dbus-python.
every thing works fine except dbus has no message it can access.

It took me just too long to realize that to make spotify status work I just needed to uninstall dbus-python.

So, I think, at least make spotify-dbus and spotify-osascript separate so that users can chose.

Or else other people like me might spend too much time figuring out why it won't work,

**Comment by ZyX-I** [comment]
@arslan2012 They are already separate, you have three variants there: just spotify (will use one or the other), spotify_dbus and spotify_apple_script. They all appear in documentation: http://powerline.readthedocs.io/en/master/configuration/segments/common.html#powerline.segments.common.players.spotify_apple_script.

**Comment by arslan2012** [comment]
I'm really sorry, I did not see that

**Comment by ZyX-I** [comment]
Though it looks like a problem in logic, I remember reading that spotify does not support dbus (not “dbus is buggy”, it is just intentionally not supported for some reason) on OS X and there really could be meant `and`.

**Comment by ZyX-I** [comment]
Fixed by #1817.
